frankenstein/cmd/frankenstein/main.go

```diff
@@ -125,15 +121,15 @@ func main() {
 
 	switch mode {
 	case distributor:
-		setupDistributor(consul, consulPrefix, chunkStore, remoteTimeout)
+		setupDistributor(consul, chunkStore, remoteTimeout)
 	case ingester:
 		cfg := local.IngesterConfig{
 			FlushCheckPeriod: flushPeriod,
 			MaxChunkAge:      maxChunkAge,
 		}
-		ingester := setupIngester(consul, consulPrefix, chunkStore, listenPort, cfg, numTokens)
+		ingester := setupIngester(consul, chunkStore, listenPort, cfg, numTokens)
 		defer ingester.Stop()
-		defer deleteIngesterConfigFromConsul(consul, consulPrefix)
+		defer frankenstein.DeleteIngesterConfigFromConsul(consul)
```

Comment:
I feel like we should have an struct which embeds the local.Ingester, and adds the various consul-related facilities.  The this line can go away. WDYT?